data/constants/keycodes/keycodes_0.0.2_basic.hjson

```diff
@@ -0,0 +1,12 @@
+{
+    "keycodes": {
+        "0x00E8": {
+            "group": "media",
```

Comment:
```suggestion
            "group": "?????",
```
We'll need to find a different group for this, given the range definitions.

Reply:
Perhaps this would be better suited to live inside the `_quantum` file instead of `_basic`, with the `0x7C77` as the keycode and a group of `quantum`?

Reply:
> We'll need to find a different group for this, given the range definitions.

I'm not quite sure what you mean. Which range?

This keycode toggles airplane mode for the host system, not the keyboard. Should I make that clearer?
So I don't think it makes sense in the quantum file/category.

Reply:
If you re-run `./util/regen.sh` it updates keycode range definitions and places all mouse keycodes within the media keycodes range as a result of the non-contiguous ordering of the group name.

Reply:
See the `IS_MEDIA_KEYCODE` generated definition in the rest of the PR -- the actual keycode range is far more inclusive with this keycode assignment.

Reply:
Ahh, I see.

I changed the keycode to `0x00C1`. Now it's right after `KC_ASSISTANT` and the range should be fine, right?

Reply:
Yep.